### DISCUSSION TITLE
Welcome to geopandas Discussions!

### DISCUSSION THREAD

**Discussion by martinfleis** [geopandas/geopandas#2700] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

 If you assume that your post will result in a clear Pull Request, try opening an issue instead.

**Comment by Mushkinkhan** [comment]
am encountered this issue mentioned below when trying to use function. plot() in juypter notebook. i have imported matplotlib.pyplot as plt, and counter check coordinates, does somebody can tell me where I am making mistake.

ValueError: 'box_aspect and fig_aspect' must be positive.

**Reply by martinfleis** [reply]
Without a reproducible example or at least some code, it is hard to help.

**Comment by wenkkkkkk** [comment]
Hi, I wonder why  ubuntu geopandas to_crs ProjError: x, y, z, and time must be same size?

**Reply by wenkkkkkk** [reply]
it was posted when I used to_crs('epsg:3857')  and  pyproj is 2.6; geopandas is 0.12, system is linux, python = 3.8;

**Reply by wenkkkkkk** [reply]
I just use pip install pyproj = 3.3. that is ok

**Comment by wenkkkkkk** [comment]
def json_to_shp(file_name):
file = os.path.splitext(file_name)[0]
data = geopandas.read_file(file_name)
data.to_file(file, driver="ESRI Shapefile", encoding='utf-8')

fiona.errors.CRSError: Invalid input to create CRS: GEOGCRS["WGS 84",DATUM["World Geodetic System 1984",ELLIPSOID["WGS 84",6378137,298.257223563,LENGTHUNIT["metre",1]]],PRIMEM["Greenwic
h",0,ANGLEUNIT["degree",0.0174532925199433]],CS[ellipsoidal,2],AXIS["geodetic latitude (Lat)",north,ORDER[1],ANGLEUNIT["degree",0.0174532925199433]],AXIS["geodetic longitude (Lon)",east
,ORDER[2],ANGLEUNIT["degree",0.0174532925199433]],USAGE[SCOPE["Horizontal component of 3D system."],AREA["World."],BBOX[-90,-180,90,180]],ID["EPSG",4326]]

what's wrong with this?
